classy-prelude/ClassyPrelude.hs

```diff
@@ -175,8 +208,11 @@ import Data.Functor
 import Control.Exception (assert)
 import Control.Exception.Enclosed
 import Control.Monad (when, unless, void, liftM, ap, forever, join, replicateM_, guard, MonadPlus (..), (=<<), (>=>), (<=<), liftM2, liftM3, liftM4, liftM5)
+import Control.Concurrent.Lifted
```

Comment:
How about instead using `import Control.Concurrent.Lifted hiding (yield)` so that we don't need to have the exports enumerated above?